go.mod

```diff
@@ -5,7 +5,7 @@ go 1.21
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.6
 	github.com/MakeNowJust/heredoc/v2 v2.0.1
-	github.com/OctopusDeploy/go-octopusdeploy/v2 v2.38.1
+	github.com/OctopusDeploy/go-octopusdeploy/v2 v2.44.2-0.20240629192925-20edc738f146
```

Comment:
NOSHIP: This makes it pick up the [unmerged go-octopusdeploy from my PR](https://github.com/OctopusDeploy/go-octopusdeploy/pull/190/files) so we can check we get a green build.

Needs to be replaced with a real version once the go-octopusdeploy PR merges and there's been a release